src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -447,7 +447,9 @@ function BaseSelectionList<TItem extends ListItem>(
                                         )}
                                     </View>
                                 )}
-                                {!headerMessage && !canSelectMultiple && customListHeader}
+                                {!headerMessage && !canSelectMultiple && Boolean(customListHeader) && (
+                                    <View style={[styles.peopleRow, styles.userSelectNone, styles.ph4, styles.pb3, listHeaderWrapperStyle]}>{customListHeader}</View>
+                                )}
```

Comment:
I don't like this change - I think styling it differently should happen in the `WorkspaceMembersPage`

Reply:
Yea, I agree with you and originally I tried to style it all in `WorkspaceMembersPage` but for some reason the header wouldn't show up on iOS/android, so I had to add this view. Let me try to explore another solution and see if that fixes the issue.